### PR TITLE
[JENKINS-67864] Table columns get wider and smaller if you click on buttons

### PR DESCRIPTION
**Pull request by pascal910107**
See [JENKINS-67864](https://issues.jenkins.io/browse/JENKINS-67864).

Before:
before
After:
after

### Proposed changelog entries

* Table columns get wider or smaller depending on the sort selection.

### Proposed upgrade guidelines

N/A

### Submitter checklist

- [x] (If applicable) Jira issue is well described
- [x] Changelog entries and upgrade guidelines are appropriate for the audience affected by the change (users or developer, depending on the change) and are in the imperative mood. [Examples](https://github.com/jenkins-infra/jenkins.io/blob/master/content/_data/changelogs/weekly.yml)
  * Fill-in the `Proposed changelog entries` section only if there are breaking changes or other changes which may require extra steps from users during the upgrade
- [x] Appropriate autotests or explanation to why this change has no tests
- [ ] New public classes, fields, and methods are annotated with `@Restricted` or have `@since TODO` Javadoc, as appropriate. 
- [ ] For dependency updates: links to external changelogs and, if possible, full diffs

### Desired reviewers

@mention

### Maintainer checklist

Before the changes are marked as `ready-for-merge`: 

- [ ] There are at least 2 approvals for the pull request and no outstanding requests for change
- [ ] Conversations in the pull request are over OR it is explicit that a reviewer does not block the change
- [ ] Changelog entries in the PR title and/or `Proposed changelog entries` are accurate, human-readable, and in the imperative mood
- [ ] Proper changelog labels are set so that the changelog can be generated automatically
- [ ] If the change needs additional upgrade steps from users, `upgrade-guide-needed` label is set and there is a `Proposed upgrade guidelines` section in the PR title. ([example](https://github.com/jenkinsci/jenkins/pull/4387))
- [ ] If it would make sense to backport the change to LTS, a Jira issue must exist, be a _Bug_ or _Improvement_, and be labeled as `lts-candidate` to be considered (see [query](https://issues.jenkins.io/issues/?filter=12146)).

<a href="https://gitpod.io/#https://github.com/jenkinsci/jenkins/pull/6702"></a>

### CODE REVIEW

**Comment by NotMyFault** [comment]
@yaroslavafenkin You removed the needs-security-review label but didn't add the approved one. Is there anything left to do?

**Review by NotMyFault — COMMENTED** [review]
The change proposed doesn't address the issue:

https://user-images.githubusercontent.com/13383509/178161908-99876a85-ccc8-438b-bff0-c4b166d9b36f.mov

**Comment by pascal910107** [comment]
> 提議的更改並未解決該問題：
> 
>  Screen.Recording.2022-07-10.at.22.48.09.mov

I used another method to try to solve this issue

File: war/src/main/less/modules/table.less
Comment (daniel-beck) [inline war/src/main/less/modules/table.less:49]: This is weird. Why does that work? Why would every `span` inside an `a` in a `th` need to be this wide? This is the kind of thing that would benefit from a comment.

**Comment by daniel-beck** [comment]
As a side effect, this increases the size of the keyboard focus. I don't know whether that's an acceptable tradeoff, but it shouldn't go unconsidered.

Screenshot 2022-07-12

**Comment by NotMyFault** [comment]
> As a side effect, this increases the size of the keyboard focus. I don't know whether that's an acceptable tradeoff, but it shouldn't go unconsidered.
> 
> Screenshot 2022-07-12

Checking out the latest master revision, I see no difference in terms of the size of keyboard navigation focus states.

**Comment by daniel-beck** [comment]
> Checking out the latest master revision, I see no difference in terms of the size of keyboard navigation focus states.

This is new in this PR.

#### weekly.ci:
Screenshot 2022-08-12 at 12 18 12

#### PR build:
Screenshot 2022-08-12 at 12 17 55

**Comment by NotMyFault** [comment]
Oh, right, absolutely. I was looking at the frame size only, in terms of tickness.

**Comment by basil** [comment]
> As a side effect, this increases the size of the keyboard focus. I don't know whether that's an acceptable tradeoff, but it shouldn't go unconsidered.

I can think of a lot of worse side effects than increasing the size of the keyboard focus. I may be naive but this seems like an acceptable tradeoff to me. Is it an unacceptable tradeoff for someone else?

**Review by basil — APPROVED** [review]

**Review by NotMyFault — APPROVED** [review]
Larger tab selectors appear to be an acceptable tradeoff to address this issue.

**Comment by basil** [comment]
This PR is now ready for merge. We will merge it after approximately 24 hours if there is no negative feedback. Please see the [merge process documentation](https://github.com/jenkinsci/jenkins/blob/master/docs/MAINTAINERS.adoc#merge-process) for more information about the merge process. Thanks!

**Comment by basil** [comment]
Manually verified as of commit 8cba74a2aa

**Pull request merged**
